src/cdn.ts

```diff
@@ -26,17 +26,19 @@ export interface RequestCDN {
 	domainName:    string;
 	/** The name of the CDN. */
 	name:          string;
+	/**
+	 * A TTL (Time To Live) value, in seconds, that, if set, overrides all set
+	 * TTL values on Delivery Services in this CDN.
+	 */
+	ttlOverride?: number;
```

Comment:
... but it _is_ allowed to be `null` in requests

Reply:
gotcha

Reply:
* Reverted making `ResponseCDN` extend `RequestCDN` in da46bce8f9
* Make ttlOverride nullable in requests but optional in responses in f778ad65b1